strawberry_django/permissions.py

```diff
@@ -187,12 +187,14 @@ def get_with_perms(
 ) -> Optional[Any]: ...
 
 
-def get_with_perms(pk, info, *, required=False, model=None):
+def get_with_perms(
+    pk, info, *, required=False, model=None, key_attr: Optional[str] = "id"
```

Comment:
I would suggest keeping the default of `pk` here, as django will automatically resolve it if you define a primary key in a column that is not named `id`

```suggestion
    pk, info, *, required=False, model=None, key_attr: Optional[str] = "pk"
```

Reply:
Also, you need to add this typing to the overloads above the implementation as well